Joplin_install_and_update.sh

```diff
@@ -127,6 +127,25 @@ print 'Downloading Joplin...'
 TEMP_DIR=$(mktemp -d)
 wget -qnv --show-progress -O ${TEMP_DIR}/Joplin.AppImage https://github.com/laurent22/joplin/releases/download/v${RELEASE_VERSION}/Joplin-${RELEASE_VERSION}.AppImage
 wget -qnv --show-progress -O ${TEMP_DIR}/joplin.png https://joplinapp.org/images/Icon512.png
+wget -qnv --show-progress -O ${TEMP_DIR}/Joplin.AppImage.sha512 https://github.com/laurent22/joplin/releases/download/v${RELEASE_VERSION}/Joplin-${RELEASE_VERSION}.AppImage.sha512
+
+#-----------------------------------------------------
+print 'Verifying download'
+if [[ -f "/usr/bin/sha512sum" ]]; then
+  HASH=$(sha512sum ${TEMP_DIR}/Joplin.AppImage | cut -f 1 -d ' ')
+  CHECKSUM=$(cat ${TEMP_DIR}/Joplin.AppImage.sha512)
+  if [[ $HASH != $CHECKSUM ]]; then
+    echo "${COLOR_RED}Checksum verification failed${COLOR_RESET}"
+    echo "Please re-run script"
+    print "Cleaning up..."
+    rm -rf $TEMP_DIR
```

Comment:
I think the script needs to be refactored a bit so that cleaning up is done within one function, which would be called from here and at the end of the script.

Reply:
On it...

Reply:
Also, one thing. Should I rebase this branch with the master branch or the dev branch? (You seem to have made a change)

Reply:
Yes dev branch please. Master is now for releases.